build.subr

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# Shell script for building Phoenix-RTOS ports
+#
+# Common functions
+#
+# Copyright 2024 Phoenix Systems
+# Author: Daniel Sawka
+#
+
```

Comment:
maybe write at the top which ENV variables are expected to be set with short decription? (or even use  bash 
```
: "${UNSET_VAR?Unset variable error!}"`
```
)

Reply:
Done